### DISCUSSION TITLE
Can't find class

### DISCUSSION THREAD

**Discussion by LaptopCat** [vfsfitvnm/frida-il2cpp-bridge#512] [Q&A] [answered]
With this code snippet:
```ts
const Game = Il2Cpp.domain.assembly("Assembly-CSharp").image
Game.class("SessionService.<>c__DisplayClass14_0").method("<ProcceedMessage>b__0").implementation = function() {
    console.log("weird")
}
```
I get this error: `couldn't find class SessionService.<>c__DisplayClass14_0 in assembly Assembly-CSharp.dll`

The class in output from `Il2Cpp.dump()`:
![image](https://github.com/vfsfitvnm/frida-il2cpp-bridge/assets/98688203/cef6181f-2c49-47ec-a7c5-3f09d15261f5)

In output from Il2CppDumper:
![image](https://github.com/vfsfitvnm/frida-il2cpp-bridge/assets/98688203/0ab622ee-2326-42d0-8649-5ce7eab38496)

**Comment by thinhbuzz** [comment, marked as answer]
Game.class("SessionService").nested('<>c__DisplayClass14_0')

**Reply by LaptopCat** [reply]
That works, thank you!
